### PR TITLE
Very minor SWAT suit changes with fresh lore.

### PR DESCRIPTION
**Pull request by kilkun**
:cl:
add: New lore surrounding the various SWAT suits.
tweak: Captain's hardsuit/SWAT suit got a few buffs. It's now much more robust.
fix: Captain's space suit is now heat proof as well as fireproof. Long overlooked no longer.
/:cl:

since deep lore is sorely sorely lacking. enjoy.

### CODE REVIEW

**Comment by ma44** [comment]
Yackemflam: _Heavy Breathing_

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -513,13 +513,14 @@
...
 /obj/item/clothing/suit/space/hardsuit/captain
-	name = "captain's hardsuit"
-	desc = "A bulky, heavy-duty piece of exclusive Nanotrasen hardsuit armor. YOU are in charge!"
+	name = "captain's SWAT suit"
+	desc = "A MK.II SWAT suit with streamlined joints and armor made out of superior materials, insulated against intense heat. The most advanced tactical armor available Usually reserved for heavy hitter corporate security, this one has a regal finish in Nanotrasen company colors. Better not let the assistants get a hold of it."
 	icon_state = "caparmor"
 	item_state = "capspacesuit"
 	allowed = list(/obj/item/weapon/tank/internals, /obj/item/device/flashlight,/obj/item/weapon/gun/energy, /obj/item/weapon/gun/ballistic, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs)
-	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
+	armor = list(melee = 80, bullet = 50, laser = 70, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
```
Comment (Cheridan) [inline code/modules/clothing/spacesuits/hardsuit.dm]: """"""very minor""""""

Comment (kilkun) [inline code/modules/clothing/spacesuits/hardsuit.dm]: assistants can still beat the shit out of the captain with toolboxes, it just takes like 3 extra hits.

there was a time when the captain's space suit was reskinned deathsquad armor. i agree that in today's day and age that might be poor taste, but the current suit is actually kind of weaksauce so I closed the gap a teensy bit.

it translates into like 3 extra toolbox hits from assistants.

Comment (kilkun) [inline code/modules/clothing/spacesuits/hardsuit.dm]: its always important to note that dr has made armor a lot less relevant than it was when it was rng to block all damage.
a 40% to 80% jump in probability to block all damage (and then if that fails, a reroll to block half damage (and then another reroll for 4/5ths damage)) is a lot more meaningful than a dam*.6 jumping to dam*.2

Comment (TechnoAlchemisto) [inline code/modules/clothing/spacesuits/hardsuit.dm]: name = "clockwork cuirass"
	desc = "A bulky cuirass made of brass."
	icon = 'icons/obj/clothing/clockwork_garb.dmi'
	icon_state = "clockwork_cuirass"
	w_class = WEIGHT_CLASS_BULKY
	body_parts_covered = CHEST|GROIN|LEGS
	resistance_flags = FIRE_PROOF | ACID_PROOF
	armor = list(melee = 80, bullet = 70, laser = -25, energy = 0, bomb = 60, bio = 0, rad = 0, fire = 100, acid = 100)

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -30,8 +30,8 @@ Contains:
 	return
 
 /obj/item/clothing/suit/space/hardsuit/deathsquad
-	name = "deathsquad suit"
-	desc = "A heavily armored, advanced space suit that protects against most forms of damage."
+	name = "MK.III SWAT Suit"
+	desc = "A prototype designed to replace the ageing MK.II SWAT suit. Based on the streamlined MK.II model, the traditional ceramic and graphene plate construction was replaced with plasteel, allowing superior armor against most threats. There's room for some kind of energy projection device on the back."
```
Comment (GunHog) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: You are not editing the correct hardsuit. See https://github.com/tgstation/tgstation/pull/21933/files#diff-6f8cb79de28d012195aecce429bafe5eR697 for the current shielded units currently in use.

**Comment by PKPenguin321** [comment]
Should be closed just on the principle of misleadingly calling them """minor""" changes

Also @optimumtact they're touching swat armor again reeee

**Comment by kilkun** [comment]
be nice to my pr titles

**Comment by Supermichael777** [comment]
ah for the days when 100 armor made you literally invulnerable  to that damage type.

**Comment by kilkun** [comment]
and swat armor had 90 bullet and 90 laser and anybody with a space suit could get it.

**Comment by KorPhaeron** [comment]
Yeah but back then tasers lasted literally 15 seconds and bypassed all armour and there was no delay to stripping it off

**Comment by kilkun** [comment]
yeah but revolver + thermals were still reliable murder machines and it was very hard to protect against flashbangs

**Comment by KorPhaeron** [comment]
You could outrun flashbangs easily

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 2 above)
Comment (ShadowDeath6) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: I think it's aging, not ageing

**Comment by optimumtact** [comment]
no armour buffs.

**Comment by kilkun** [comment]
@optimumtact fixed

**Comment by optimumtact** [comment]
The buffs are still there, are you trying to play games iwth me young man

**Comment by optimumtact** [comment]
ah okay it was github not updating the view

**Review by optimumtact — APPROVED** [review]

**Comment by kilkun** [comment]
yay it turns out im not as gitretarded as i thought i was

**Comment by KingPhilipIII** [comment]
Honestly, I think the Captain needs to stop getting buffs. He already has a recharging laser and a sword, the latter being incredibly deadly for a ceremonial thrusting weapon, and now we're buffing his armor.

**Comment by kilkun** [comment]
to be fair he's also the fuckin' captain and should be a force to be reckoned with. there's also only one of them.

though the saber is kind of bullshit.

**Comment by KingPhilipIII** [comment]
It's not so much an issue with this particular kind of change. Just the trend I've observed. 

It's annoying when I go in to kill the Captain with a double energy sword and he takes my arm off with one swipe from his saber because logic and then guns me down with the laser when I grab the sword and try to haul ass.

Has happened to me twice now. Once ending with him shooting me. Another with him taking off a leg then disembowling me. There's a difference between defending oneself and leading the charge against enemies armed with your trusty sword and a taser you've liberated from the armory because you're the captain.

**Comment by TechnoAlchemisto** [comment]
The captain is supposed to be hard to kill

**Comment by Bawhoppen** [comment]
WAS THIS A TEST TO SEE IF I WASN'T ACTIVE AT THE TIME BECAUSE I LEARNED NOW HAHAHA BITCHES ITS TIME EAT SHIT :^):^):^):^):^):^):^):^):^):^):^):^):^)

**Comment by Bawhoppen** [comment]
help me

**Comment by dannno** [comment]
aging not ageing

**Comment by kilkun** [comment]
what are you upset about?

**Pull request merged**